src/luascript.cpp

```diff
@@ -2656,6 +2654,8 @@ void LuaScriptInterface::registerFunctions()
 	registerMethod("Player", "getGuildNick", LuaScriptInterface::luaPlayerGetGuildNick);
 	registerMethod("Player", "setGuildNick", LuaScriptInterface::luaPlayerSetGuildNick);
 
+	registerMethod("Player", "isInWar", LuaScriptInterface::luaPlayerIsInWar);
```

Comment:
ARE AT WAR

please fix this longstanding mistake

Reply:
Fixed